app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import time
+from datetime import datetime
+
+
+def create_file_every_second():
+    while True:
+        file_name = f"app-{datetime.now().hour}_{datetime.now().minute}_" \
+                    f"{datetime.now().second}.log"
+        file_content = str(datetime.now())
```

Comment:
You call datetime.now() four times
It's better to store this value in a separate variable and use it.